tests/Opus/Validate/IssnTest.php

```diff
@@ -0,0 +1,129 @@
+<?php
+/**
+ * This file is part of OPUS. The software OPUS has been originally developed
+ * at the University of Stuttgart with funding from the German Research Net,
+ * the Federal Department of Higher Education and Research and the Ministry
+ * of Science, Research and the Arts of the State of Baden-Wuerttemberg.
+ *
+ * OPUS 4 is a complete rewrite of the original OPUS software and was developed
+ * by the Stuttgart University Library, the Library Service Center
+ * Baden-Wuerttemberg, the Cooperative Library Network Berlin-Brandenburg,
+ * the Saarland University and State Library, the Saxon State Library -
+ * Dresden State and University Library, the Bielefeld University Library and
+ * the University Library of Hamburg University of Technology with funding from
+ * the German Research Foundation and the European Regional Development Fund.
+ *
+ * LICENCE
+ * OPUS is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU General Public License as published by the Free Software
+ * Foundation; either version 2 of the Licence, or any later version.
+ * OPUS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details. You should have received a copy of the GNU General Public License
+ * along with OPUS; if not, write to the Free Software Foundation, Inc., 51
+ * Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ *
+ * @category    Tests
+ * @package     Opus_Validate
+ * @author      Maximilian Salomon (salomom@zib.de)
+ * @copyright   Copyright (c) 2017, OPUS 4 development team
+ * @license     http://www.gnu.org/licenses/gpl.html General Public License
+ */
+
+
+/**
+ * Unittests for Opus_Validate_Issn.
+ * @coversDefaultClass Opus_Validate_Issn
+ */
+class Opus_Validate_IssnTest extends TestCase
+{
+    /**
+     * Overwrite parent methods.
+     */
+    public function setUp() {}
```

Comment:
Ich weiß, dass das bei anderen Tests im Framework auch so gemacht wird. Es ist aber eine schlechte Idee. Der Gedanke hier ist, dass die Tests in dieser Klasse kein spezielles Setup bzw. Teardown benötigen. Da sie unabhängig von der Datenbank sind, ist es egal, ob die Datenbank vor den Tests geleert wird oder nicht. Das sollte dann aber auch im Kommentar erklärt werden. Das eigentliche Problem ist aber, dass SetUp und TearDown in der Zukunft durchaus Aufgaben erfüllen könnten, die wirklich für alle Tests wichtig sind. Die Funktionalität der übergeordneten Klassen sollte nicht einfach ausgeschaltet werden.

Der bessere Ansatz wären zwei unterschiedliche Basisklassen für normale Tests und für Datenbanktests. Dieses Refactoring machen wir später mal. Hier einfach nur die leeren Funktionen löschen auch wenn die Tests dadurch erst einmal langsamer werden.